plasmapy/atomic/tests/test_atomic.py

```diff
@@ -1286,3 +1290,382 @@ def test_extract_integer_charge_warnings(test_input, expected_warning):
     """Test that _extract_integer_charge issues the expected warnings."""
     with pytest.warns(expected_warning):
         _extract_integer_charge(test_input)
+
+
+# argument, expected
+period_table = [
+    ("ca", 4),
```

Comment:
If we use `atomic_symbol`, then these tests would need to be changed to be case-sensitive.  The element names are not case-sensitive, though.